### PR TITLE
New package: Livebook.Livebook version 0.9.3

### PR DESCRIPTION
**Pull request by sarvex**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110827&drop=dogfoodAlpha

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
Hmmm, It look like that is weird,. It matched with the Sigcheck and other hashes application tools but why there is Validation-Hash-Verification-Failed.

**Comment by CoolPlayLin** [comment]
But, The value of SHA-256 is correct
image

**Comment by sarvex** [comment]
I am not sure what is going on here. @somebody #help

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installation failed with exit code -1978335215 
> 2023-06-28 04:14:01.561 [CLI ] Package hash verification failed. 
> SHA256 in manifest [dd26b3f54d22207b253e0c01531a974b0cab91ac4e6c7cb907ebdadc471c9082] does not match download [889f174a5458ea7393b0b980014d089cf2c68977d60ed281f25970b60d8e305f] 

This might be due to a data center IP block, to prevent automated downloads.

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 8:46:38 AM:  > 2023-07-05 08:46:37.940 [CLI ] Package hash verification failed. SHA256 in manifest [dd26b3f54d22207b253e0c01531a974b0cab91ac4e6c7cb907ebdadc471c9082] does not match download [889f174a5458ea7393b0b980014d089cf2c68977d60ed281f25970b60d8e305f]

**Comment by sarvex** [comment]
@stephengillie again SHA mismatch :(

File: manifests/l/Livebook/Livebook/0.9.3/Livebook.Livebook.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Livebook.Livebook
+PackageVersion: 0.9.3
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://livebook.dev/releases/latest/LivebookInstall-latest-windows-x86_64.exe
+  InstallerSha256: 889F174A5458EA7393B0B980014D089CF2C68977D60ED281F25970B60D8E305F
```
Comment (CoolPlayLin) [inline manifests/l/Livebook/Livebook/0.9.3/Livebook.Livebook.installer.yaml:10]:
```suggestion
  InstallerSha256: DD26B3F54D22207B253E0C01531A974B0CAB91AC4E6C7CB907EBDADC471C9082
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
image

File: manifests/l/Livebook/Livebook/0.9.3/Livebook.Livebook.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Livebook.Livebook
+PackageVersion: 0.9.3
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://livebook.dev/releases/latest/LivebookInstall-latest-windows-x86_64.exe
```
Comment (CoolPlayLin) [inline manifests/l/Livebook/Livebook/0.9.3/Livebook.Livebook.installer.yaml:9]:
```suggestion
  InstallerUrl: https://github.com/livebook-dev/livebook/releases/download/v0.9.3/LivebookInstall-v0.9.3-windows-x86_64.exe
```

Comment (CoolPlayLin) [inline manifests/l/Livebook/Livebook/0.9.3/Livebook.Livebook.installer.yaml:9]:
```suggestion
  InstallerUrl: https://github.com/livebook-dev/livebook/releases/download/v0.9.3/LivebookInstall-v0.9.3-windows-x86_64.exe
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
Please do not use `latest` instead of specific version

**Review by stephengillie — APPROVED** [review]

**Pull request merged**